server/routes/user.js

```diff
@@ -1,10 +1,54 @@
 const router = require('express').Router();
-const { User } = require('../db');
+const { User } = require('../db').models;
 
-function check(req, res) {
-  res.send('It works!!');
+const DEFAULT_LIMIT = 30;
+
+function create(req, res, next) {
+  return User.create(req.body)
+    .then(user => res.send(user))
+    .catch(err => next(err));
+}
+
+function get(req, res, next) {
+  const id = req.params.id;
+
+  return User.findById(id)
+    .then(user => res.send(user))
+    .catch(err => next(err));
+}
+
+function list(req, res, next) {
+  const limit = Number(req.query.limit) || DEFAULT_LIMIT;
+  const offset = Number(req.query.offset) || 0;
```

Comment:
We already have this in `Device` router so it is good moment to convert it to small express middleware that can be applied either globally or before selected routes.

Reply:
@vladimyr, what would be exact purpose i.e. functionality of such middleware?

Reply:
Lettuce-man make it easy for me so I'll just proxy you to: https://github.com/expressjs/express-paginate :grin:

Reply:
I agree with this, but don't do it now, let's leave it for a separate branch and apply it to all routers, not just `/user`.